content/applications/productivity/discuss/chatter.rst

```diff
@@ -0,0 +1,416 @@
+
+Click :guilabel:`📎 Attachments` to add any files to the message, then click :guilabel:`Send`.
+
+.. image:: chatter/chatter-full-composer.png
+   :align: center
+   :alt: The expanded full chatter composer in the CRM application.
+
+Edit sent messages
+------------------
+
+Messages can be edited after they are sent, to fix typos, correct mistakes, or add missing
+information.
+
+.. note::
+   When messages are edited after they have been sent, an updated message is **not** sent to the
+   recipient.
+
+To edit a sent message, click the :guilabel:`... (ellipsis)` icon menu to the right of the message.
+Then, select :guilabel:`Edit`. Make any necessary adjustments to the message.
+
+.. image:: chatter/chatter-edit.png
+   :align: center
+   :alt: The edit message option in a chatter thread.
+
+To save the changes, press :command:`Ctrl + Enter`. To discard the changes, press :command:`Escape`.
+
+.. important::
+   Users with Admin-level access right can edit any sent messages. Users without Admin rights can
+   **only** edit messages they created.
+
+.. _discuss/search-messages:
+
+Search messages
+===============
+
+Chatter threads can become long after a while, because of all the information they contain. To make
+it easier to find a specific entry, users can search the text of messages and notes for specific
+keywords.
+
+Select a record with a chatter thread. For example, to search a *CRM* opportunity, navigate to
+:menuselection:`CRM app --> Sales --> My Pipeline` and click on the Kanban card of an opportunity to
+open it. Then, at the top-right above the chatter composer, click the :guilabel:`🔍 (magnifying
+glass)` icon to open the search bar.
+
+Enter a keyword or phrase into the search bar, then hit enter, or click the :guilabel:`🔍
+(magnifying glass)` icon to the right of the search bar. Any messages or notes containing the
+keyword or phrase entered are listed below the search bar with the keyword highlighted.
+
+To be taken directly to a particular message in the chatter thread, hover over the upper-right
+corner of the result to reveal a :guilabel:`Jump` button. Click this button to be directed to that
+message's location in the thread.
+
+.. figure:: chatter/chatter-search.png
+   :align: center
+   :alt: Search results in a chatter thread emphasising the search icon and the jump button.
+
+   Search results in a chatter thread. Hover over the upper right corner of a result to see the Jump
+   option. Click it to be taken directly to that message in the chatter thread.
+
+.. _discuss/schedule-activities:
+
+Schedule activities
+===================
+
+*Activities* are follow-up tasks tied to a record in an Odoo database. Activities can be scheduled
+on any database page that contains a chatter thread, Kanban view, list view, or activities view of
+an application.
+
+To schedule an activity through a chatter thread, click the :guilabel:`Schedule Activity` button,
```

Comment:
```suggestion
To schedule an activity through a chatter thread, click the :guilabel:`Activities` button,
```
I'm seeing it as just 'Activities' in the 17.0 runbot I'm using for the review. might be worth a quick double-check during your revisions to make sure